shamir/tables.go

```diff
@@ -0,0 +1,77 @@
+package shamir
```

Comment:
This is forked from [Vault](https://github.com/hashicorp/vault/tree/master/shamir) and does not need to be reviewed.